Azure Support management SDK for stable version "2020-04-01"

shahbj79 opened the pull request:
- This change generates Azure Support management SDK against API version 2020-04-01.
- Tests have been added for all the APIs.
- Here is the link to API [Spec](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/support/resource-manager/Microsoft.Support/stable/2020-04-01)

Commit:
support sdk and tests

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
addressing review comments

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
sdk for stable version

Commit:
Merge branch 'master' of https://github.com/Azure/azure-sdk-for-net

isra-fel commented on sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj:
```diff
@@ -6,12 +6,12 @@
     <PackageId>Microsoft.Azure.Management.Support</PackageId>
     <Description>Azure Support SDK allows creation and management of Azure support tickets. Using the operations for support case management, you can (1) Open a technical, billing, subscription management, or quota increase support ticket, (2) Get list and detailed information about support tickets, (3) Update severity, contact information of a support ticket (4) Add a new communication and get l ...
-    <Version>1.0.0-preview.1</Version>
+    <Version>1.0.0.0</Version>
```
```suggestion
    <Version>1.0.0</Version>
```
Please use 3-digit for project versioning

isra-fel reviewed (changes requested):
Hi, I noticed that the PR is missing a change to /eng/mgmt/mgmtmetadata/hybriddatamanager_resource-manager.txt. I suspect it's caused by running autorest directly, rather that our recommended way: running src/generate.ps1 in your module.

So, please do the following things:

- Edit src/generate.ps1, change -AutoRestVersion "latest" to "2.0.4413"
- Run it to generate SDK
    - In case you encounter any error, please refer to our document: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk
- Make sure you check in the txt file mentioned above
- By the way, please put link(s) to your API spec review PR in the description

Thanks

Commit:
addressing review comments

shahbj79 commented:
> Hi, I noticed that the PR is missing a change to /eng/mgmt/mgmtmetadata/hybriddatamanager_resource-manager.txt. I suspect it's caused by running autorest directly, rather that our recommended way: running src/generate.ps1 in your module.
> 
> So, please do the following things:
> 
> * Edit src/generate.ps1, change -AutoRestVersion "latest" to "2.0.4413"
> * Run it to generate SDK
>   
>   * In case you encounter any error, please refer to our document: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk
> * Make sure you check in the txt file mentioned above
> * By the way, please put link(s) to your API spec review PR in the description
> 
> Thanks

**Addressed all comments.**
**Here is the link to API [spec ]**(https://github.com/Azure/azure-rest-api-specs/tree/master/specification/support/resource-manager/Microsoft.Support/stable/2020-04-01)

isra-fel reviewed (approved):
LGTM
